resources/definitions/fdmprinter.def.json

```diff
@@ -4424,6 +4424,19 @@
                     "settable_per_mesh": false,
                     "settable_per_extruder": true
                 },
+                "prime_tower_purge_volume":
+                {
+                    "label": "Prime Tower Purge Volume",
+                    "description": "Amount of filament to be purged when wiping on the prime tower. Purging is useful for compensating the filament lost by oozing during inactivity of the nozzle.",
+                    "type": "float",
+                    "enabled": "dual_pre_wipe",
```

Comment:
Whether a setting is enabled is independent of whether it's true...

You should still make this depend on both.